src/main/java/org/actviti/cloud/audit/JpaAuditApplication.java

```diff
@@ -52,8 +52,10 @@ public void run(String... strings) throws Exception {
     }
 
     @StreamListener(AuditConsumerChannels.AUDIT_CONSUMER)
-    public synchronized void receive(ProcessEngineEventEntity event) {
-        eventsRepository.save(event);
+    public synchronized void receive(ProcessEngineEventEntity[] events) {
```

Comment:
See my related comment in https://github.com/Activiti/Activiti/pull/1424: could you use `List` here instead of array?